demos/pedestrian_tracker_demo/cpp/main.cpp

```diff
@@ -224,8 +299,6 @@ int main(int argc, char **argv) {
                 PrintDetectionLog(log);
         }
 
-        slog::info << "Metrics report:" << slog::endl;
-        metrics.logTotal();
```

Comment:
Undo the deletion of lines introduced in develop branch (and lines 210-211, 191-192).

Reply:
done

Reply:
`presenter.drawGraphs(frame);`
`metrics.update(startTime, frame, { 10, 22 }, cv::FONT_HERSHEY_COMPLEX, 0.65);`
these lines should be located after **drawing tracked detections** part, otherwise drawing time is not measured and included in performance metrics. Please, put it in a correct place.